projects/plugins/protect/src/js/components/threats-list/paid-list.jsx

```diff
@@ -184,7 +191,12 @@ const PaidList = ( { list } ) => {
 						description,
 						diff,
 						filename,
+						firstDetected, // todo: still needs a proper fix
+						first_detected,
 						fixedIn,
+						fixed_in,
+						fixedOn,
+						fixed_on,
```

Comment:
For some reason I can't seem to uncover, on the initial load these three properties are in `camelCase` and on every other re-render they are converted to `snake_case`. This was valid option to resolve the issue of those values being `undefined` when they were not in the expected format, but I am thinking a more permanent solution should be applied.

Reply:
I think there is a casing conversion function in the redux store somewhere. I'll try to find it.